From 93dbaec3de36195e80629ca4a793dd07b1dcaa34 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 19 Feb 2018 17:25:24 +0900
Subject: [PATCH 597/909] arm64: dts: r8a77990: Add MFIS Lock device node

commit a3e3649edfa400ff6575f15384bd87f69f300eeb from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MFIS Lock device node to the R8A77990 SoC.

NOTE: Since the locking device needs to start earlier than any device,
      it is described at the top of the device tree as a special case.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index c671d1e8acb6..146bc330311c 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -188,6 +188,13 @@
 		#size-cells = <2>;
 		ranges;
 
+		mfis: mfis@e6260000 {
+			compatible = "renesas,mfis-lock-r8a77990",
+				     "renesas,mfis-lock";
+			reg = <0 0xe6260000 0 0x1000>;
+			#hwlock-cells = <1>;
+		};
+
 		rwdt: watchdog@e6020000 {
 			compatible = "renesas,r8a77990-wdt",
 				     "renesas,rcar-gen3-wdt";
-- 
2.17.1

